### DISCUSSION TITLE
After I installing pyg-2.0.2 using conda or pip, I can't import MLP from from 'torch_geometric.nn.models'

### DISCUSSION THREAD

**Discussion by WMQ1999** [pyg-team/pytorch_geometric#3680] [Q&A] [answered]
Just like the pic:
![2021-12-11 17-12-02 的屏幕截图](https://user-images.githubusercontent.com/34160344/145671226-9a318bd3-71cf-4e69-aa4d-afc3e3c4717e.png)

**Comment by rusty1s** [comment, marked as answer]
The `MLP` model just got recently added. You need to install PyG from source to make use of it:
```
pip install git+https://github.com/pyg-team/pytorch_geometric.git
```

**Reply by WMQ1999** [reply]
Thank you very much, it worked!
